benches/mercat_account.rs

```diff
@@ -38,9 +37,9 @@ fn bench_account_creation(c: &mut Criterion) {
         &label,
         move |b, (_label, secret_account)| {
             b.iter(|| {
-                let account_creator = AccountCreator {};
+                let account_creator = AccountCreator;
```

Comment:
do we really need this can't we directly do something like this - 

```
AccountCreator.create(&secret_account, &valid_asset_ids, &mut rng).unwrap();
```

Reply:
Other places have similar changes

Reply:
we can. this is only done in tests, so I didn't bother.